net/dnsdist/Makefile

```diff
@@ -25,6 +24,35 @@ PKG_CONFIG_DEPENDS:= \
 
 include $(INCLUDE_DIR)/package.mk
 
+define Package/dnsdist/Default
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=IP Addresses and Names
+  TITLE:=dnsdist DNS-, DOS- and abuse-aware loadbalancer $(2)
+  USERID:=dnsdist:dnsdist
+  DEPENDS:= \
+	  +ca-bundle \
+	  +libatomic \
+	  +libcap \
+	  +libstdcpp \
+	  +luajit
+  URL:=https://dnsdist.org/
+  VARIANT:=$(1)
+  PROVIDES:=dnsdist
+endef
+
+define Package/dnsdist/description/Default
+  dnsdist is a highly DNS-, DoS- and abuse-aware loadbalancer. Its goal in life
+  is to route traffic to the best server, delivering top performance to legitimate
+  users while shunting or blocking abusive traffic.
+endef
+
+define Package/dnsdist/conffiles/Default
+/etc/dnsdist.conf
+/etc/config/dnsdist
+/etc/init.d/dnsdist
+endef
+
 define Package/dnsdist/config
```

Comment:
I think, it would be good to move this to the ``Config.in`` as it is getting very long.

Reply:
That makes complete sense to me, will do! Thanks!

Reply:
Done!